src/index.scss

```diff
@@ -1,9 +1,19 @@
-$feature-flags: (
```

Comment:
The `feature-flags` mapping should not be removed. Your layout is not rendering properly without it: https://carbon-tutorial-pfine.mybluemix.net/#/

Reply:
thanks, fixed and re-deployed :D